assets/js/modules/analytics/components/common/AccountSelect.js

```diff
@@ -67,10 +69,18 @@ export default function AccountSelect() {
 			const newAccountID = item.dataset.value;
 			if ( accountID !== newAccountID ) {
 				selectAccount( newAccountID );
-				trackEvent( 'analytics_setup', 'account_change', newAccountID );
+				const action =
+					newAccountID === ACCOUNT_CREATE
+						? 'change_account_new'
+						: 'change_account';
+				trackEvent(
+					`${ viewContext }_analytics`,
+					action,
+					newAccountID
```

Comment:
Event label is not needed for this event.

Reply:
https://github.com/google/site-kit-wp/pull/4321/commits/2af228490d3eda940db320070394ea8c8c406b8a